.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+executors:
+  node:
+    docker:
+      - image: cimg/node:14.17.6
+    working_directory: ~/app
+  docker_builder:
+    docker:
+      - image: circleci/buildpack-deps:bionic
+        user: root
+
+references:
+  nuxt_parameters: &nuxt_parameters
+    NUXT_APP_NODE_URL:
+        type: string
+    NUXT_APP_NODE_WS:
+      type: string
+    NUXT_APP_NETWORK_NAME:
+      type: string
+    NUXT_APP_FAUCET_API:
+      type: string
+    NUXT_APP_ENABLE_FAUCET:
+      type: string
+  docker_parameters: &docker_parameters
+    DOCKERHUB_REPO:
+        type: string
+    VERSION_TAG:
+        type: string
+  
+commands:
+  npm_build:
+    parameters: *nuxt_parameters
+    steps:
+      - checkout
+      - run:
+          name: Install
+          command: npm install
+      - run:
+          name: Generate
+          command: NUXT_APP_ENABLE_FAUCET="<< parameters.NUXT_APP_ENABLE_FAUCET >>" NUXT_APP_FAUCET_API="<< parameters.NUXT_APP_FAUCET_API >>" NUXT_APP_NODE_URL="<< parameters.NUXT_APP_NODE_URL >>" NUXT_APP_NODE_WS="<< parameters.NUXT_APP_NODE_WS >>" NUXT_APP_NETWORK_NAME="<< parameters.NUXT_APP_NETWORK_NAME >>" npm run generate
```

Comment:
Multiline would look much better, one var per line

Reply:
done in docker build command, because we decided to move the npm build in the Dockerfile.